src/event_listener.cc

```diff
@@ -59,6 +59,18 @@ const std::string compressType2String(const rocksdb::CompressionType type) {
   return iter->second;
 }
 
+bool isDiskQuotaExceeded(const rocksdb::Status &bg_error) {
+    // EDQUOT: Disk quota exceeded (POSIX.1-2001)
+    std::string edquot_str = "Disk quota exceeded";
+    std::string err_msg = bg_error.ToString();
+
+    if (err_msg.length() < edquot_str.length()) {
+        return false;
+    }
+
+    return edquot_str == err_msg.substr(err_msg.size()-edquot_str.size());
```

Comment:
can use string find API?

Reply:
Yes, we can use find. The reason I use tail matching is to know that it can only be at the tail, which is probably a little bit faster than find.
It doesn't seem necessary. I will use find API.

Reply:
Yes, background error wasn't the hot path, so it will be good to make it more readable.